### ISSUE TITLE
Loop on resizing with percentage width stacks

### ISSUE THREAD

**Issue by jasonrclark** [shoes/shoes4#926]
Reported originally at http://librelist.com/browser//shoes/2014/10/20/shoes-4-loops-uninterruptibly-after-resize-when-stack-width-is-stated-as-a-percentage/

Given this program:

```ruby
#!    /usr/bin/env jruby
require 'shoes'

Shoes.app do
   flow do
     stack(width: '50%') do
       button "One"
     end
     stack(width: '50%') do
       button "Two"
     end
   end
end
```

Grab the bottom of the window and drag it up, resizing the window, as 
far as possible, then drag it down again. Notice the black area when you 
pull it down. Shoes loops uninterruptibly when this happens.

Running on jruby 1.7.16 (1.9.3p392) 2014-09-25 575b395 on OpenJDK 64-Bit Server VM 
1.7.0_65-b32 +jit [linux-amd64]
Built using rvm.
OS is Debian Jessie, which is a pre-release.

Some images of the problem, which we haven't entirely duplicated yet:

Just after starting the app: http://algoram.com/tmp/shoes/1.png
Bottom pulled up as far as it will go: http://algoram.com/tmp/shoes/2.png
Pull it down again, and oops, it’s looping and not responding: 
http://algoram.com/tmp/shoes/3.png

**Comment by jasonrclark** [comment]
Pinged mailing list about this, as the recently merged #1024 might give us some diagnostics, but I'm unable to repro it on OS X.

**Comment by PragTob** [comment]
We don't have feedback --> closing until we get some
